.github/workflows/test_mlagility.yml

```diff
@@ -58,6 +58,8 @@ jobs:
           benchit examples/cli/scripts/two_models.py
```

Comment:
Increment version number?

Reply:
Saving it for #227

Reply:
see https://github.com/groq/mlagility/pull/227/files#r1152032887